### PR TITLE
add test package and corrected bug

### PR DESCRIPTION
**Pull request by emilienschultz**
I added a tests directory, a test file and a travis file. I packaged it (Matthias did) with flit

### CODE REVIEW

File: flit.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[metadata]
+module = sempervirens
+author = Nathaniel Smith
+author-email = njs@vorpus.org
+home-page = https://github.com/Carreau/sempervirens
```
Comment (arokem) [inline flit.ini]: Should be on `njsmith/sempervirens`?

Comment (Carreau) [inline flit.ini]: Yep, that's my fault. Autogenerated :-)

Comment (emilienschultz) [inline flit.ini]: fixed

File: flit.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[metadata]
+module = sempervirens
+author = Nathaniel Smith
+author-email = njs@vorpus.org
```
Comment (Carreau) [inline flit.ini]: And my fault also it is `njs@pobox.com` be we can change that later.

Comment (emilienschultz) [inline flit.ini]: fixed

**Pull request closed without merging**